src/chapter1/challenges.md

```diff
@@ -16,12 +16,25 @@ To get setup:
 - Run:
 
 ```sh
-git clone <repo-link>                                   # Clone to your machine
-cd <repo-name>                                          # Enter clone's directory
-git remote add upstream <repo-link>                     # Create link to template called 'upstream' 
-git remote set-url --push upstream DISABLE              # Disable pushing to template
-git fetch upstream                                      # Sync with 'upstream'
-git merge upstream/main --allow-unrelated-histories     # Merge 'upstream' main branch with your main
+# Clone to your machine
+git clone <repo-link>
+
+# Enter clone's directory
+cd <repo-name>
+
+# Create link to template called 'upstream' 
+git remote add upstream https://github.com/MonashDeepNeuron/HPC-Training-Challenges.git
```

Comment:
Do we want to have a separate note for SSH users or are we enforcing the use of HTTP?

Reply:
SSH requires a more elaborate setup with creating key-pairs between you local machine and GitHub so I assume if they know how to use SSH and want to use it they know how to get the SSH link.